tests/test_barostat.py

```diff
@@ -323,19 +323,19 @@ def test_barostat_varying_pressure():
         coords.shape[0], pressure, temperature, group_indices, barostat_interval, u_impls, seed, True, 0.0
     )
 
-    ctxt = custom_ops.Context(coords, v_0, complex_box, integrator_impl, u_impls, movers=[baro])
-    ctxt.multiple_steps(1000)
+    ctxt = custom_ops.Context(coords, v_0, box, integrator_impl, u_impls, movers=[baro])
+    ctxt.multiple_steps(3000)
```

Comment:
Change needed for convergence?

Reply:
Change was convergence, though I ended up changing the test it looks like. Will adjust back to the way it was as we don't need to run the extra MD.

Reply:
Fixed up in https://github.com/proteneer/timemachine/pull/1339/commits/d0b8f7c3fa1fe94fb68b4a3192830efefbc1a407